src/views/Signup/components/SignupForm.tsx

```diff
@@ -204,6 +214,20 @@ function SignUpForm(): JSX.Element {
 
     handleVPNAuth(isVPNAuth, xNewToken);
 
+    if (isOAuthFlow) {
+      const user = localStorageService.getUser();
+      if (user) {
+        const success = handleOAuthSuccess(user, xToken, xNewToken);
```

Comment:
if `xToken` is not longer used (I think we just use `xNewToken` as far as I know), maybe we can remove it.

Reply:
I'll remove it from here, guys, as I hadn't considered that it's no longer in use. Good catch :)